scripts/run_whitesource.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+#
+# Run whitesource for components which need versioning
+
+# SCAN_DIRECTORIES is a comma-separated list (as a string) of file paths which contain all source code and build artifacts for this project
+echo "PWD:"${PWD}
+SCAN_DIRECTORIES=${PWD}
+echo "SCAN_DIRECTORIES:"$SCAN_DIRECTORIES
+
+
+if [[ "$PWD" == *travis* ]]; then
```

Comment:
https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
```
if [[ -n "$TRAVIS"]]; then
```
This is more reliable.

Reply:
Will do!

Reply:
See https://github.com/snowflakedb/snowflake-connector-python/blob/master/ci/wss.sh#L45-L54 fo Travis checks.

I don't think TRAVIS_PULL_REQUEST becomes an empty if not PR.

Reply:
Yes, I saw my build using false as a value. Will change it, thank you!
Sorry I couldnt have verified this before generating a PR. (Unless a draft)